### PR TITLE
Vikas branch

### PR DESCRIPTION
**Pull request by vikaspatil29**
Added new changes in test.py

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+kjhdjchd
```
Comment (sqilup) [inline test.py]: Code is junk pleas

**Comment by haidharma** [comment]
please do it quickly next time

File: test.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+dsclkdkc
+dcvdeded
```
Comment (haidharma) [inline test.py]: junk...

File: test.py (hunk 1 above)
Comment (haidharma) [inline test.py]: baddd

**Review by sqilup — APPROVED** [review]

**Pull request merged**